csaf_2.1/language_specific_translation/translations.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/csaf/master/csaf_2.1/test/language_specific_translation/translations_json_schema.json",
+  "translation_version": "2.1",
+  "translations": [
+    {
+      "lang": "de",
+      "term": {
+        "reasoning_for_withdrawal": "Begründung für die Zurückziehung"
+      }
+    }
+  ]
```

Comment:
We can also do:
```
  "translation": 
    {
      "de": {
        "reasoning_for_withdrawal": "Begründung für die Zurückziehung"
      }
    }
```

However, I guess we need either specify all accepted languages (adapt the schema each time - which makes it harder for all that use the JSON as they might need to update their program if we add keys...) or use `patternProperties`. The latter is definitely less work on the schema, but not sure were that leaves us with the implementations :thinking:

Reply:
Could we not use a pattern in the JSON schema for these keys?

Reply:
> Could we not use a pattern in the JSON schema for these keys?

Sorry - just to get this right: Do you want to use the `patternProperties` for the JSON keys?

Reply:
Yes, that was my thinking.

Reply:
Done in [bcc250b](https://github.com/oasis-tcs/csaf/pull/901/commits/bcc250b4194d8ebee1fceeb411e1d94f7c5fe74c)